### DISCUSSION TITLE
Add Term "Call" as 「呼叫」

### DISCUSSION THREAD

**Discussion by cschan1828** [python/python-docs-zh-tw#484] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

call

### 術語翻譯

呼叫

### 說明

Python 文件常見單字之一，應明確定義為「呼叫」，以維持文件一致性。

### 參考資料

- [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html)中將 Callable 翻譯為「可呼叫物件」
- [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html)中將 Callback 翻譯為「回呼」
- 國家教育研究院樂詞網將 Call 翻譯為 「呼叫」

**Comment by mattwang44** [comment]
for reference, 我對於翻譯 `invoke` 一詞的看法
https://github.com/python/python-docs-zh-tw/pull/469#discussion_r1260916275
